components/Dashboard/ReviewForm.tsx

```diff
@@ -62,6 +62,30 @@ const ReviewForm: React.FC<ReviewFormProps> = (props) => {
     }
   }
 
+  const paragraphs = introMsg.map((paragraph) => {
+    const trimmedParagraph = paragraph?.trim()
+    let paragraphElements
+    if (trimmedParagraph?.[0] === '-') {
+      paragraphElements = (
+        <>
+          <ul>
+            <li className="flex gap-2 text-base text-gray-700">
+              &bull;
+              <p className="">{trimmedParagraph?.split('-')?.[1]}</p>
+            </li>
+          </ul>
+        </>
+      )
+    } else {
+      paragraphElements = (
+        <>
+          <p className="mb-3 text-gray-500">{paragraph}</p>
+        </>
+      )
+    }
+    return <>{paragraphElements}</>
+  })
+
```

Comment:
Good Work 😄
One suggestion: Can we wrap this in the `useMemo` hook?
Also, I assume that `<ul>` renders the bullet tags by default, why did we added `&bull;` here?